docs/pages/develop/user-interface/assets.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Assets
+description: Learn about using static assets in your project, including images, videos, sounds, database files, and fonts.
+---
+
+Files such as images, videos, sounds, database files for SQLite, or fonts are considered **static assets**. They are not part of your project's JavaScript but are part of your app. These assets can be served locally from your project or remotely over the network.
+
+## Serve an asset locally
+
+When an asset is stored in your project's file system, it can be embedded in your app binary at build time or loaded at runtime. You can import them like JavaScript modules using `require` or `import` statements.
+
+For example, to render an image called **example.png** in **App.js**, you can directly use `require` to import the image from the project's **assets/images** directory and pass it to the `<Image>` component:
+
+```jsx App.js
+<Image source={require('./assets/images/example.png')} />
+```
+
+The bundler automatically provides a width and height for the images imported, as shown in the above example. For more information, see [Static Image Resources](https://reactnative.dev/docs/images#static-image-resources).
+
+Libraries such as `expo-image` and `expo-file-system` work similarly to the `<Image>` component with local assets.
+
+### How are assets served locally
+
+Locally stored assets are served over HTTP in development. They are automatically bundled into your app binary at the build time for production apps and served from disk on a device.
+
+### Load an asset at build time
+
+> **Note:** The `expo-asset` config plugin is only available for SDK 51 and above. If you are using an older SDK, you can load a [using the `useAssets` hook](/versions/v51.0.0/sdk/asset/#useassetsmoduleids).
+
+Install the `expo-asset` library and add the config plugin to the app config file. This plugin will embed the asset file in your native project.
+
+```json app.json
+{
+  "plugins": [
+    [
+      "expo-asset",
+
+      {
+        "assets": ["./assets/images/example.png"]
+      }
+    ]
+  ]
+}
+```
+
+The `assets` option takes an array of one or more asset files or directory names to link the file to the native project. The path to each file is relative to the project's root.
+
+When you [create a new native build](/develop/development-builds/create-a-build/), you can import and use the asset in your project without using a `require` or an `import` statement.
+
+For example, the **example.png** is linked by the above config plugin. You can directly import it into your component:
+
+```jsx App.js
+import { Image } from 'expo-image';
+/* @hide ... */ /* @end */
+
+function App() {
+  return <Image source="example" />;
```

Comment:
we might want to mention that this isn't particularly useful, but if some native api expects a resource name then this makes integration with that api easier

Reply:
I've added the following explanation below the example.

![CleanShot 2024-05-07 at 04 55 46@2x](https://github.com/expo/expo/assets/10234615/d3530e0c-99b5-4e8a-876f-243f6ae0a282)